M2/Macaulay2/packages/PencilsOfQuadrics.m2

```diff
@@ -700,11 +728,13 @@ randomLineBundle(ZZ, RingElement) := (d,f) -> (
 	);
     Ld
     )
-///
---restart
---load "PencilsOfQuadrics.m2"
-///
-///--test of preRandomLineBundle and randomLineBundle-- but there were no "asserts"
+
+
+TEST///
+--test of preRandomLineBundle and randomLineBundle
+restart
+load "PencilsOfQuadrics.m2"
+>>>>>>> 82c56807936dda91b86f9ccae7189b001e15ff19
```

Comment:
This line is going to cause errors (and also the "restart" two lines above will prevent this test from running)